### PR TITLE
Query : Get Account List

### PR DESCRIPTION
**Pull request by parv0888**
## Purpose
Added support for `GetAccountList`.

## Changes
Added support for `GetAccountList`.

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [x] (If necessary) I have updated the CHANGELOG.

## CLA acceptance
By submitting the contribution I accept the terms and conditions of the
Contributor License Agreement v1.0
- link: https://developers.concordium.com/CLAs/Contributor-License-Agreement-v1.0.pdf

- [x] I accept the above linked CLA.

Depends on #84

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -373,6 +379,12 @@ val peers = client.getPeerList(true);
 SemVer version = client.getVersion();
 ```
 
+#### getAccountList
+```java
+val accounts = client
```
Comment (EmilLa1) [inline README.md:384]: Please provide the type - not everyone uses `Lombok`.

**Review by EmilLa1 — CHANGES_REQUESTED** [review]
Looks good. 
Please provide a unit test.

File: README.md (hunk 1 above)
Comment (parv0888) [reply README.md]: done

**Review by EmilLa1 — APPROVED** [review]
Looks good

**Pull request merged**